src/vde_router/vde_headers.h

```diff
@@ -22,6 +22,15 @@
 #define PROTO_TCP 6
 #define PROTO_UDP 17
 
+#define XSTR(x) STR(x)
+#define STR(x) #x
+
+#ifdef VDE_DARWIN
```

Comment:
I think this was here for debug purposes. Can we get rid of it?

Reply:
Oh, sure, yes! I'm sorry I forgot to delete those lines :(
Do you want to do it yourself or do you prefer I do it and resubmit the pull request?

Reply:
simply commit on top of the branch, the PR gets updated automatically.